src/System.ServiceModel.Primitives/ref/System.ServiceModel.Primitives.cs

```diff
@@ -1450,7 +1450,7 @@ public sealed partial class WindowsClientCredential
         public System.Net.NetworkCredential ClientCredential { get { return default(System.Net.NetworkCredential); } set { } }
     }
     public sealed partial class X509ServiceCertificateAuthentication {
-        internal X509ServiceCertificateAuthentication() { }
+        public X509ServiceCertificateAuthentication() { }
```

Comment:
Isn't this an API addition? This will probably need to go through and API review See https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/api-review-process.md